EIPS/eip-x.md

```diff
@@ -0,0 +1,113 @@
+---
+eip: x
+title: CREATE3
+author: Moody Salem
+category: Core
+type: Standards Track
+status: Draft
+created: 2020-12-22
+---
+
+### Specification
+
+This EIP is a fork of [./eip-1014.md](EIP-1014) which adds an account creation opcode that does not involve the init code.
+
+Adds a new opcode (`CREATE3`) at `0xf6`, which takes 4 stack arguments: endowment, memory_start, memory_length, salt. 
+Behaves identically to `CREATE2` (`0xf5`), except using `keccak256( 0xfe ++ address ++ salt )[12:]`,
+which does not include the hash of the init code.
+
+The `CREATE3` has the same `gas` schema as `CREATE2`, but with one fewer `GSHA3WORD` opcode, to account for the difference
+in lengths of the preimages.
+The `hashcost` is deducted at the same time as memory-expansion gas and `CreateGas` is deducted: _before_ evaluation
+of the resulting address and the execution of `init_code`.
+
+- `0xfe` is a single byte, 
+- `address` is always `20` bytes, 
+- `salt` is always `32` bytes (a stack item). 
+
+The preimage for the final hashing round is thus always exactly `53` bytes long (does not include hash of the init code).
+
+### Motivation
+
+The core motivations are the same as [./eip-1014.md](EIP-1014), i.e. deploying a smart contract at a deterministic address.
+This variation removes the hash of the init code from the preimage. The issue with the existing `CREATE2` opcode
+is that computing the hash of the init code on-chain requires the entire bytecode of the target contract to be included
+in the contract computing the first preimage.
+
+### Rationale
+
+#### Address formula
+
+* As with CREATE2, ensures that addresses created with this scheme cannot collide with addresses created using the traditional `keccak256(rlp([sender, nonce]))` formula, as `0xfe` can only be a starting byte for RLP for data many petabytes long.
+* The addresses created with this scheme cannot collide with addresses created using the `CREATE2`, as `0xfe` will never collide with `0xff`.
+* Ensures that the hash preimage has a fixed size.
+
+#### Gas cost
+
+The concerns for `CREATE2` gas cost in regard to init code preimage length do not apply.
+
+### Clarifications
+
+This EIP makes collisions possible. The behaviour at collisions is specified by [EIP-684](https://github.com/ethereum/EIPs/issues/684):
```

Comment:
EIP-684 is not an EIP, just an idea.  I don't know what the current on-chain behavior is since 1014 is a valid final EIP and it references 684, but regardless, you should not reference non-final EIPs in an EIP (note: some older EIPs don't follow this rule as it is relatively new).